draft-ietf-quic-qpack.md

```diff
@@ -550,18 +550,39 @@ the block.  Blocking decoders use the Largest Reference to determine when it is
 safe to process the rest of the block.
 
 `Base Index` is used to resolve references in the dynamic table as described in
-{{indexing}}.  To save space, Base Index is encoded relative to Largest
-Reference using a one-bit sign flag.
+{{indexing}}.
 
-    baseIndex = largestReference + deltaBaseIndex
+To save space, Base Index is encoded relative to Largest Reference using a
+one-bit sign and the `Delta Base Index` value.  A sign bit of 0 indicates that
+the Base Index has a greater absolute index than the Largest Reference; the
```

Comment:
greater than or equal to, technically.  We don't mention the zero case at all until the bottom.

Reply:
Well the sentence says that the encoder added entries.  I don't think that it's a problem to not mention the zero case.

Reply:
I don't see where this sentence says it added entries - are you referring to somewhere else?  I think Mike's right here - if the Base Index is equal to LR then the sign bit will be 0.  The original text had that, and it's included on lines 578-9 below.  Is it clearer if we bring it back up here?

I agree we don't need to mention the case where they are both 0 here.